devices/ikea.js

```diff
@@ -552,7 +552,7 @@ module.exports = [
         extend: tradfriExtend.light_onoff_brightness(),
     },
     {
-        zigbeeModel: ['SILVERGLANS IP44 LED driver'],
+        zigbeeModel: ['SILVERGLANS IP44 LED driver', 'Pendant lamp WW'],
```

Comment:
Since the device has a separate model, it can be distinguished from this led driver. Can you move it into a new definition?